s:1224:"<b>Test 1: Teste 1 </b><br />Incorrect program result
<br /> --- Input ---
<br /><pre><i> 6
1
2
3
4
5
6
7
6
5
4
3
2
9
8
8
8
9
1
</i></pre>
<br /> --- Program output ---
<br /><pre><i>Digite a quantidade de elementos da lista:Digite o elemento da lista:Digite o elemento da lista:Digite o elemento da lista:Digite o elemento da lista:Digite o elemento da lista:Digite o elemento da lista:Digite o elemento da lista:Digite o elemento da lista:Digite o elemento da lista:Digite o elemento da lista:Digite o elemento da lista:Digite o elemento da lista:Digite o elemento da lista:Digite o elemento da lista:Digite o elemento da lista:Digite o elemento da lista:Digite o elemento da lista:Digite o elemento da lista:Traceback (most recent call last):
  File "funcoes1.py", line 37, in &lt;module&gt;
    if crescente(len(a)-1):
  File "funcoes1.py", line 6, in crescente
    for i in range(0,len(lista)-1,1):
TypeError: object of type 'int' has no len()
</i></pre>
<br /> --- Expected output (text)---
<br /><pre><i> 'S'
'N'
'N'
'N'
'S'
'N'
'N'
'N'
'S'
</i></pre>
<br /><b>Summary of tests</b><br /><pre><i>+------------------------------+
|  1  test run/ 0 tests passed |
+------------------------------+
</i></pre>
<br /><br />";